app/views/static_pages/community_rules.html.erb

```diff
@@ -13,166 +13,151 @@
-      </li>
-      <li>
-        <strong>Unsolicited pings/DM/friend request</strong>
-      </li>
-      <li>
-        <strong>Discussing unrelated topics that can be divisive or harmful</strong> such as politics, religion, relationships, mental health, medical conditions, medicine, homeopathic or other home remedies, etc.
-      </li>
-      <li>
-        <strong>Dog piling/short circuiting a teaching moment</strong> by jumping in to help when someone else is already helping out or giving an answer when someone is leading someone to think critically about their problem.
-      </li>
-      <li>
-        <strong>Chat bombing</strong> by posting many short (usually 4-5 or fewer words) messages in rapid succession or repeatedly sending parts of thoughts (incomplete, non-standalone sentences or fragments) in new messages. Instead, use shift + enter and send one message where possible or edit your message.
-      </li>
-      <li>
-        <strong>Inappropriate User Profile</strong> with suggestive or inappropriate memes as profile picture, username/nickname, or bio.
-      </li>
-      <li>
-        <strong>Crossposting or Duplicate Posts</strong> by posting the same message to different channels or asking for help in multiple channels at once. You should allow at least <strong>30 minutes</strong> for a reply. While you are waiting, make sure that you posted in the most relevant channel, provided a <a href="/how_to_ask" target="_blank">detailed question</a>, and continue trouble-shooting. It is recommended to edit your post with the results of things you have tried. If your post gets buried, you may repost your question, otherwise, you may direct people to your question in a more active channel.
-      </li>
-      <li>
-        <strong>No fundraising or donation drives</strong> while we're very sympathetic to variety of causes, moderating the authenticity of such initiatives is not something we can commit to. Please find other communities that allow such activities, including sharing links to this kind of content.
-      </li>
+      <li>Discuss topics that are safe and welcoming for everyone</li>
+      <li>Ask questions about content or projects in our curriculum</li>
+      <li>Ask questions about the tools recommended in our curriculum</li>
+      <li>Ask questions in public channels for anyone to answer</li>
+      <li>Ask your question in only one channel</li>
+      <li>Help others by guiding them to the solution in a public 1:1 conversation</li>
+      <li>Make it easy for others to help you by asking a detailed question</li>
+      <li>Share resources that are relevant to our curriculum</li>
+      <li>Report misconduct to ModMail</li>
+      <li>Treat everyone with respect</li>
     </ul>
 
-    <h3 id="zap-level-1">1 Zap - Warning for Minor Violations</h3>
-
+    <h3 id="dont-summary"><span class="fa-solid fa-triangle-exclamation mr-1"></span> Don't</h3>
```

Comment:
Same as above but using amber for the color instead. Not 100 sold on the color but I think some variation of yellow would be better than red. Red just seemed too hostile when I tried it lol

```suggestion
    <h3 id="do-summary" class="flex items-center"><span class="fa-solid fa-triangle-exclamation mr-3 text-lg text-amber-600"></span> Don't</h3>
```

Outputs:
<img width="980" alt="Screenshot 2023-03-28 at 09 01 54" src="https://user-images.githubusercontent.com/7963776/228169561-7049bbba-621b-4606-b4dd-41315b0acb84.png">